platooning_strategic_IHP/include/platoon_manager_ihp.h

```diff
@@ -131,6 +137,7 @@ namespace platoon_strategic_ihp
         * 
         * \param senderId static id of the broadcasting vehicle
         * \param platoonId platoon id
+        * \param senderBsmId bsm id of the broadcasting vehicle
```

Comment:
Description of BSM ID has been added, but I don't see an actual parameter that this refers to.  We don't really want this, do we?